core/ssh_keys.go

```diff
@@ -0,0 +1,106 @@
+package core
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/krystal/go-katapult"
+)
+
+type AuthSSHKey struct {
+	ID          string `json:"id,omitempty"`
+	Name        string `json:"name,omitempty"`
+	Fingerprint string `json:"fingerprint,omitempty"`
+}
+
+type SSHKeysClient struct {
```

Comment:
The `SSHKeysClient` struct needs to be added to `core.Client` in `core/client.go`.